### ISSUE TITLE
Typo in codebase

### ISSUE THREAD

**Issue by srivats22** [flutter/flutter#181153]
In ```semantics.dart``` line 122 AccessiblityFocusBlockType the spelling of Accessiblity is incorrect and should be: AccessibilityFocusBlockType.

This will be just a Name change across all files and tests, with no functional impact based on my analysis.

I am currently working on another issue, but I can take this up after I finish my other work.

Edit:

Might be a breaking change cause if someone has defined a semantic widget and is already using this then they would need to refactor the codebase after the upgrade

**Comment by badri001** [comment]
may I pick this one

**Comment by tirth-patel-nc** [comment]
Variable is not present in 3.38.7 but present in latest master 3.41.0-1.0.pre-164. Hence variable/typo was introduced after 3.38.7

**Comment by Piinks** [comment]
If someone would like to send a PR, we'd be happy to take it! :) If not, let us know and we can fix it right quick.

**Comment by srivats22** [comment]
Would this be considered a breaking change?

If so should I handle it differently??

I am ready with the change but just want to ensure not breaking anything
